server/app.js

```diff
@@ -9,17 +9,22 @@ app.use(express.json());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-
 app.use('/api/v1', usersRouter);
 app.use('/api/v1', articleRouter);
 app.use('/api/v1', comment);
-app.use((req, _res, next) => {
+app.get('/', (_req, res) => {
+  let message = ' Welcome to Team work. ';
+  message += 'The goal of this application is to facilitate more interaction between colleagues and facilitate team bonding.';
+  res.send(message);
+});
+
+app.use((_req, _res, next) => {
   const error = new Error('Bad url ');
   error.status = 404;
   next(error);
 });
 
-app.use((error, req, res, next) => {
+app.use((error, _req, res) => {
```

Comment:
Why did you change `req` to `_req`

Reply:
Es lint showed me the error on that parameter. In trying to fix that with using esLint hints I choose `prefix all unused declaration with '_'  where possible`,  
And then after this happened and has no impact on the application workflow.